_docs/releasing_version.md

```diff
@@ -6,69 +6,100 @@ summary: How to perform a complete version release, including modulesync and pub
 ---
 
 Creating a release is a two step process:
-1. Prepare the release — setup everything so that peer review can happen and when everything is ready…
+1. Prepare the release
 2. Do the actual release
 
 ## Preparing a release
 
-Run modulesync to ensure the dotfiles are up-to-date.
+Before preparing a release, it may help to run modulesync to ensure the
+dotfiles are up-to-date. If modulesync has been run recently by someone else,
+you may not have to do it. See https://github.com/voxpupuli/modulesync for more
+info on using modulesync.
 
-Create a 'release pr'. This pull request updates the changelog and bumps the
-version number to the target version, removing all release candidate
-identifiers, i.e. from `0.10.7-rc0` to `0.10.7`. Here's an example:
-[puppet-extlib's 0.10.7 release](https://github.com/voxpupuli/puppet-extlib/pull/43).
-In most cases it is sufficient to update metadata.json. We try
-to respect [semantic versioning](http://semver.org/) and decided that dropping ruby1.8
-support is a major change and requires a major version bump for the module.
-(Only the minor version should be bumped if the module is pre version 1.0 and
-ruby 1.8 support has been dropped.)
-
-If necessary, run `bundle install` before continuing. If you want you can also only install the needed gems:
+### 1. Create a new branch for a "release pull request"
 
+For example, if you want to release version `1.2.3` of a module:
 ```bash
-bundle install --path .vendor/ --without system_tests development
+git checkout -b "release_123"
 ```
 
-And in case you installed the gems before:
+### 2. Bump the version
+
+Bump the version number in the module's `metadata.json`.
+
+The version bump must remove all release candidate identifiers and move to the
+next appropriate release version. For example: `0.10.7-rc0` to `0.10.7`.
+
+Here's a Pull Request to use as a reference:
+[puppet-extlib's 0.10.7 release](https://github.com/voxpupuli/puppet-extlib/pull/43).
+
+Please follow [semantic versioning](http://semver.org/) when changing the
+version number.
+
+### 3. Update the CHANGELOG
+
+After bumping the version in `metadata.json`, use the following rake task to
+automatically update the CHANGELOG.
+
+NOTE: This requires a [GitHub access token (docs on how to create
+one)](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
+The changelog generator expects the token in the environment variable
+`CHANGELOG_GITHUB_TOKEN`
+
+> If necessary, run `bundle install` before continuing.
+> ```bash
+> bundle install --path .vendor/ --without system_tests development
+> ```
+>
+> And in case you installed the gems before:
+> ```bash
+> bundle install --path .vendor/ --without system_tests development; bundle update; bundle clean
+> ```
```

Comment:
I wonder if this should be simplified to just always doing the install/update/clean command, so that you don't have to know the answer to when it would be necessary to run `bundle install` before continuing.

Reply:
that's the reason why I initially provided it as a long line. just copy paste it and it will set everything up.